server/models/file.coffee

```diff
@@ -70,14 +70,21 @@ File.createNew = (fileName, path, url, tags, callback) ->
         class: getFileClass mime
         mime: mime
 
+    # Clear the temporary file
+    clearTmpFile = (cb) ->
+        log.info "Deleting file: #{filePath}"
+        fs.unlink filePath, (err) ->
+            callback err, null if err?
+            cb()
```

Comment:
Is that normal to call bath callback and cb if there is an error ? Shouldn't we call cb(err) ?